python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -104,3 +108,23 @@ def __init__(self, num_processes: int = 1,
 
             super().__init__(accelerator=accelerator,
                              plugins=[plugin], *args, **kwargs)
+
+    @staticmethod
+    def compile(model: nn.Module, loss: _Loss = None, optimizer: torch.optim = None):
+        """
+        Compile a pytorch model into a pytorch-lightning model and return it.
+        Args:
```

Comment:
maybe write this in the format as https://github.com/intel-analytics/BigDL/blob/branch-2.0/python/chronos/src/bigdl/chronos/autots/tspipeline.py#L47-L60

Reply:
Applied your suggestion. Thanks.